Classes/Mapper/ContentRepositoryMapper.php

```diff
@@ -92,60 +87,52 @@ public function initializeObject()
     /**
      * @param Asset $asset
      * @param MetaDataCollection $metaDataCollection
-     * @throws NodeTypeNotFoundException
-     * @throws \TYPO3\Flow\Persistence\Exception\IllegalObjectTypeException
+     *
      * @return void
+     * @throws NodeTypeNotFoundException
```

Comment:
In Neos we don't annotate nested exception. So these annotations can be removed.